src/lib/features/personal-dashboard/personal-dashboard-controller.e2e.test.ts

```diff
@@ -253,9 +281,9 @@ test('should return personal dashboard project details', async () => {
                 ),
             },
             {
-                createdBy: 'audit user',
+                createdBy: 'unknown',
```

Comment:
wondering what this change is about

Reply:
Honestly, I don't know why `createdBy` is "unknown" here, but it feels like it's further down the stack and I didn't wanna start tangling with that now. We could remove that altogether and slice the array, though. to only get the flag events. 🤔  But since we're checking the whole object, maybe not.

Reply:
I added previous one because it was there in the test result. But I'm surprised that it has changed. Let's keep it for now and observe this test. If it becomes unstable because of this we'll act accordingly.

Reply:
Oh, it changed because I added the user to the project after creating it, which triggers another event. That's why the event's changed. I don't think it'll be unstable. But we'll see.